API_v1.md

```diff
@@ -689,15 +927,15 @@ Create a new user with a place and a contact.
 
 ### Permissions
 
-`can_create_users`  
+`can_create_users`, `cat_create_places`, `cat_create_contacts`
```

Comment:
Which role should allow these permissions? `national_admin` and `district_admin`?